avm/res/hybrid-compute/machine/tests/e2e/max.hci/main.test.bicep

```
@@ -0,0 +1,92 @@
+targetScope = 'subscription'
+
+metadata name = 'Creates an Arc Machine with maximum configurations'
+metadata description = 'This instance deploys the module with the full set of required parameters.'
```

```suggestion
metadata description = 'This instance deploys the module with most of its features enabled.'
```

The rationale behind this terminology is that the required parameter must be deployed by the default test already - but the max test (or any other test for that matter) will deploy **more** than just the default :)